### DISCUSSION TITLE
[BUG] subtask input element not always cleared

### DISCUSSION THREAD

**Discussion by jojoob** [TechbeeAT/jtxBoard#830] [General]
**Describe the bug**
When adding a subtask by pressing enter on the keyboard instead of clicking the add button after inserting the summary/title (either via the "quick add subtask" or subtasks element) the text input box is not cleared when the text has a trailing space. Just the trailing space will be removed. But the subtasks will be added with the trailing space. 

**To Reproduce**
1. Go to a task or create one
2. Type some text **with a trailing space** into the "quick add subtasks" or "add a subtask" field 
3. Press enter on the (virtual) keyboard 
4. See the input field not cleared but the trailing space removed. And the subtask added with a trailing space

**Expected behavior**
The input field should be cleared when the input has a trailing space like it is when the input has no trailing space. Additionally trailing spaces could be removed from the subtask summary as they often happen due to virtual keyboards adding auto spaces after word prediction/correction. 

**Device and version:**
 - Device: Samsung galaxy s10e
 - Android version 12
 - jtx Board version: 2.05.03.gplay (205030010) 
 - Downloaded from: Google Play

**Comment by patrickunterwegs** [comment]
Hi @jojoob , that's a bit weird, I can't really reproduce this issue. Do you use any specific keyboard? Tonight I will also check if this might be a specific issue on Android 12

**Comment by jojoob** [comment]
> Do you use any specific keyboard?

Yes, that's it. It only happens with the Microsoft SwiftKey Keyboard.

**Comment by patrickunterwegs** [comment]
Thanks for the update! 
I don't know why such keyboard issues pop up from time to time, it seems to be related to specific keyboards and UI elements in jetpack compose (which is the most modern way to make an app in Android anyway) But a lot of apps are still using XML layouts that don't seem to have this issue. That's why people feel like it's an issue in jtx Board while in reality it's the keyboard in combination with the standard UI element...

So regrettably there's nothing I can do here. I'll move this to discussions to keep it visible for other users.
